marshal/message.go

```diff
@@ -233,3 +247,25 @@ func (m *msgClaimingMessages) Type() msgType {
 func (m *msgClaimingMessages) Timestamp() int {
 	return m.Time
 }
+
+// msgReleaseGroup is used by the Admin to pause a consumer group,
+// identified by groupID, until MsgExpireTime.
+type msgReleaseGroup struct {
+	msgBase
+	MsgExpireTime int
```

Comment:
Semantically yeah. I actually don't expose this to the user, they only get to control the pause timeout duration.